### PR TITLE
Readability improvements.

### PR DESCRIPTION
**Pull request by mertcandav**
Readability improvements for ``./doc/``

## Validation Steps Performed
- [x] CLA signed.

### CODE REVIEW

File: doc/WindowsTestPasses.md [hunk 1]
```diff
@@ -26,10 +28,10 @@ Prerequisites:
 
 1. Open Visual Studio 2017 and use the TDP drop-down menu to open the `Device Manager`.
 1. In the pane that opens to the left, choose `Add` and then `Nebula VM Device`. Nebula is a cloud provider for VMs (like Azure but a more private instance for corporate work usage).
-1. Name the machine and choose the build/branch/flavor/SKU from the drop downs at the bottom. It will find the VHD for you from the build shares. Hit `Add Device` to deploy to Nebula.
+1. Name the machine and choose the ``ild/branch/flavor/SKU``rom the drop downs at the bottom. It will find the VHD for you from the build shares. Hit `Add Device` to deploy to Nebula.
```
Comment (WSLUser) [inline doc/WindowsTestPasses.md:31]: This is actually less readable than before. Please switch this back.

File: doc/terminal-v1-roadmap.md [hunk 2]
```diff
@@ -90,6 +100,6 @@ Feature Notes:
 
 \* Feature Priorities:
 
-0. Mandatory <br/>
-1. Optimal <br/>
-2. Optional / Stretch-goal <br/>
++ Mandatory <br/>
```
Comment (WSLUser) [inline doc/terminal-v1-roadmap.md:103]: The numbers denote the priority level. Add them back.

File: doc/terminal-v2-roadmap.md [hunk 3]
```diff
@@ -74,9 +81,9 @@ Feature Notes:
 
 \* Feature Priorities:
 
-0. Mandatory <br/>
-1. Optimal <br/>
-2. Optional / Stretch-goal <br/>
++ Mandatory <br/>
```
Comment (WSLUser) [inline doc/terminal-v2-roadmap.md:84]: Here too

**Comment by WSLUser** [comment]
Keep in mind, unless it's a spec or Roadmap most of the documentation has been moved to https://github.com/microsoftdocs/terminal

**Comment by mertcandav** [comment]
Thanks for the answer, I replaced it with the old one.

File: doc/WindowsTestPasses.md [hunk 4]
```diff
@@ -26,10 +28,10 @@ Prerequisites:
 
 1. Open Visual Studio 2017 and use the TDP drop-down menu to open the `Device Manager`.
 1. In the pane that opens to the left, choose `Add` and then `Nebula VM Device`. Nebula is a cloud provider for VMs (like Azure but a more private instance for corporate work usage).
-1. Name the machine and choose the build/branch/flavor/SKU from the drop downs at the bottom. It will find the VHD for you from the build shares. Hit `Add Device` to deploy to Nebula.
+1. Name the machine and choose the build/branch/flavor/SKU rom the drop downs at the bottom. It will find the VHD for you from the build shares. Hit `Add Device` to deploy to Nebula.
```
Comment (DHowett) [inline doc/WindowsTestPasses.md:31]: This change confuses me.

Comment (mertcandav) [reply doc/WindowsTestPasses.md:31]: Oh sorry. I must have accidentally deleted a character. I'll fix it.

**Comment by j4james** [comment]
No doubt this is subjective, but I personally find the introduction of all those `<br>` breaks _less_ readable. If you're looking at the raw markdown (which is often the case for me), then they're just distracting. And in the rendered view they introduce what I'd consider a lot of unnecessary whitespace.

It seems to me that the amount of space to include before a heading should really be a stylistic choice for the renderer. Anyone that prefers more spacing could always use a different viewer, or adjust the css themselves. So I don't think it's appropriate to force a particular style on everyone by hacking in `<br>` elements everywhere when they aren't strictly necessary.

**Comment by DHowett** [comment]
@mertcandav Thanks for the contribution!

I'm inclined to agree with James on this one -- the spacing around the headings and the sections is something that should be handled on the client side... because there's a great number of different markdown renderers, with different rendering characteristics, I'm averse to introducing hard line breaks to serve just one of them.

I personally find them noisy as well, as I tend to read the plaintext versions of the documents as that's how they pass around in our review cycles and live in our working trees.

We can also ignore the entire user-docs/ folder. It's not long for this world.

**Comment by mertcandav** [comment]
Thanks for the comments!

**Pull request closed without merging**